komodor/README.md

```diff
@@ -0,0 +1,68 @@
+# Komodor - Kubernetes Troubleshooting Platform
+
+## Overview
+
+This integration allows Datadog Monitor Alerts to be available in the Komodor Kubernetes troubleshooting platform and to suggest related services based on service connections detected by Datadog.
+
+Komodor tracks changes across your entire K8s stack, analyzes their ripple effect and provides you with the context you need to troubleshoot efficiently and independently.
+
+### Empower on-call teams
+Make the knowledge and expertise that has traditionally been held by only a few, clear and visible to developers, SRE and support teams.
+
+### Understand your kubernetes
+Gain the K8s visibility you are lacking. See your deployments on a timeline with the relevant information: what changed, what code was pushed and by whom.
+
+### Track your system end-to-end
+View data from your Git, config, infra, alerting and other tools, in one centralized and easy-to-understand display, easily link to Datadog metrics with dynamic deployment links that take you directly to the data you need.
+
+### Uncovering the context
```

Comment:
To keep this section brief, would it be possible to combine the content of the "Empower on-call teams" and the content of "Increase performance and availability" with this section? I think that these sections are all related as they highlight the value of your platform, but are all higher level in terms of the actual integration.